### DISCUSSION TITLE
Does freemocap work with black and white cameras?

### DISCUSSION THREAD

**Discussion by mjhmilla** [freemocap/freemocap#209] [Q&A]
Does freemocap work with black and white cameras?

**Comment by jonmatthis** [comment]
I think so! The specific question is whether `mediapipe` (the method we're currently using to track the 2d skeletons in the camera images) will work on black and white videos. I'm pretty sure the answer is  'Yes' but I've never tried it myself. 

Easiest way to test is to follow the instructions laid out here - https://github.com/freemocap/freemocap/blob/main/OLD_README.md

...and then when you get the `Setup` phase, check the 'Mediapipe overlay' button and see if you're being picked up in the images of whatever cameras you're using. 

I'm curious why you're asking though! Are you planning to use infrared cameras or something?

**Reply by mjhmilla** [reply]
Thank you for the detailed response, I really appreciate it. I also really enjoyed your DW2022 presentation - I couldn't be there this year, but was very glad to see your work.

In regards to cameras, the cameras we have in the lab I currently work in are black and white. But of course, if the system works with black and white images then infrared is possible, and maybe a depth camera (?) could be added.
